### ISSUE TITLE
HH Member period of absence not validated

### ISSUE THREAD

**Issue by daiClegg** [dbrownukk/EFD_v2#300]
Instance: EFD_HM
Study: test Config Questions
HH Name: 3 test absence questions
HH Number: 7
- The 6th HH member has  invalid number of months absent (13)

HH Number:  8
- The 3rd HH member has  invalid number of months absent (13)

Expected Behaviour:
- Period away in months should be blank or between 0 and 12 (integer)

**Comment by dbrownukk** [comment]
added constraint on HHM - period of absence between 0 and 12

**Comment by daiClegg** [comment]
Tested this with the same HH as above.
It catches the 13 month absence in the spreadsheet

But, it leaves the app in the new Household view. That was a bit surprising.

I returned to the list view and re-selected the HH; it's marked uploaded (and unparsed).
This means i have to go back to the s'sheet to correct the invalid absence length. 
Would be nice if it passed Parse  (ie the dataset is structurally complete) but failed Validation - then i could just fix it in the app. wdyt?
